src/Microsoft.Health.Fhir.Api/Controllers/AadSmartOnFhirProxyController.cs

```diff
@@ -81,7 +81,7 @@ public AadSmartOnFhirProxyController(IOptions<SecurityConfiguration> securityCon
             {
                 if (ex is HttpRequestException || ex is OperationCanceledException)
                 {
-                    logger.LogWarning(ex, $"There was an exception while attempting to read the OpenId Configuration from \"{openIdConfigurationUrl}\".");
+                    logger.LogWarning(ex, "There was an exception while attempting to read the OpenId Configuration from \"{0}\".", openIdConfigurationUrl);
```

Comment:
Shouldn't this be as below? From what I've seen this would use the variable name `0`

```suggestion
                    logger.LogWarning(ex, "There was an exception while attempting to read the OpenId Configuration from \"{openIdConfigurationUrl}\".", openIdConfigurationUrl);
```

Reply:
TIL.
I was under impression they just do string.Format, but that's not the case. 
Updated.

---
In reply to: [610157238](https://github.com/microsoft/fhir-server/pull/1833#discussion_r610157238) [](ancestors = 610157238)